src/Razor/src/Microsoft.NET.Sdk.Razor/ValidateStaticWebAssetsUniquePaths.cs

```diff
@@ -0,0 +1,84 @@
+﻿// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+using System;
+using System.Collections.Generic;
+using System.IO;
+using Microsoft.Build.Framework;
+using Microsoft.Build.Utilities;
+
+namespace Microsoft.AspNetCore.Razor.Tasks
+{
+    public class ValidateStaticWebAssetsUniquePaths : Task
+    {
+        private const string BasePath = "BasePath";
+        private const string RelativePath = "RelativePath";
+        private const string TargetPath = "TargetPath";
+
+        [Required]
```

Comment:
Does this throw if a project has no assets?

Reply:
Same for `WebRootFiles`

Reply:
No, it simply passes an empty array and works fine.